scripts/snpseq/st2client-snpseq.sh

```diff
@@ -17,7 +17,7 @@ st2 run packs.load packs=snpseq_packs register=all
 
 ## Make virtualenv for snpseq_packs
 st2 run packs.setup_virtualenv packs=snpseq_packs
-
+/opt/stackstorm/virtualenvs/snpseq_packs/bin/pip install cryptography==38.0.4
```

Comment:
Nice! Could you add a comment that explains why this is done? Perhaps only refer to https://levelup.gitconnected.com/fix-attributeerror-module-lib-has-no-attribute-openssl-521a35d83769

Reply:
```suggestion
/opt/stackstorm/virtualenvs/snpseq_packs/bin/pip install cryptography==38.0.4 #Refer: https://levelup.gitconnected.com/fix-attributeerror-module-lib-has-no-attribute-openssl-521a35d83769
```